bodhi/server/models.py

```diff
@@ -923,6 +933,7 @@ class Update(Base):
     status = Column(UpdateStatus.db_type(),
                     default=UpdateStatus.pending,
                     nullable=False)
+    ci_status = Column(UpdateCiStatus.db_type(), default=None, nullable=True)
```

Comment:
I think you're right, thought I prefer to be explicit, I'm fine with removing this if you want.